manifests/0000_26_capi-operator_11_deployment.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-capi-operator
+  namespace: openshift-cluster-api
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/delete: "true"
+  labels:
+    k8s-app: cluster-capi-operator
+spec:
+  selector:
+    matchLabels:
+      k8s-app: cluster-capi-operator
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        k8s-app: cluster-capi-operator
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: cluster-capi-operator
+      containers:
+      - name: cluster-capi-operator
+        image: quay.io/openshift/cluster-capi-operator
+        command:
+        - ./cluster-capi-operator
+        env:
+        - name: RELEASE_VERSION
+          value: "0.0.1-snapshot"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+        volumeMounts:
+        - name: images
+          mountPath: /etc/cluster-api-config/
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      restartPolicy: Always
+      tolerations:
+      - key: node.cloudprovider.kubernetes.io/uninitialized
+        effect: NoSchedule
+        value: "true"
```

Comment:
I don't think we need to tolerate uninitialised nodes, that would imply that our pod can come up before the CCM has done its job, at this point, there would be no pod network which would block this scheduling. We should remove this toleration